Extension/package.json

```diff
@@ -282,2546 +282,2593 @@
+            "scope": "resource",
+            "minimum": 0.1,
+            "maximum": 1
+          },
+          "C_Cpp.inactiveRegionForegroundColor": {
+            "type": "string",
+            "description": "%c_cpp.configuration.inactiveRegionForegroundColor.description%",
+            "scope": "resource"
+          },
+          "C_Cpp.inactiveRegionBackgroundColor": {
+            "type": "string",
+            "description": "%c_cpp.configuration.inactiveRegionBackgroundColor.description%",
+            "scope": "resource"
+          },
+          "C_Cpp.loggingLevel": {
+            "type": "string",
+            "enum": [
+              "None",
+              "Error",
+              "Warning",
+              "Information",
+              "Debug"
+            ],
+            "default": "Error",
+            "markdownDescription": "%c_cpp.configuration.loggingLevel.markdownDescription%",
+            "scope": "window"
+          },
+          "C_Cpp.autoAddFileAssociations": {
+            "type": "boolean",
+            "default": true,
+            "markdownDescription": "%c_cpp.configuration.autoAddFileAssociations.markdownDescription%",
+            "scope": "window"
+          },
+          "C_Cpp.workspaceParsingPriority": {
+            "type": "string",
+            "enum": [
+              "highest",
+              "high",
+              "medium",
+              "low"
+            ],
+            "default": "highest",
+            "markdownDescription": "%c_cpp.configuration.workspaceParsingPriority.markdownDescription%",
+            "scope": "window"
+          },
+          "C_Cpp.workspaceSymbols": {
+            "type": "string",
+            "enum": [
+              "All",
+              "Just My Code"
+            ],
+            "default": "Just My Code",
+            "description": "%c_cpp.configuration.workspaceSymbols.description%",
+            "scope": "window"
+          },
+          "C_Cpp.exclusionPolicy": {
+            "type": "string",
+            "enum": [
+              "checkFolders",
+              "checkFilesAndFolders"
+            ],
+            "default": "checkFolders",
+            "markdownDescription": "%c_cpp.configuration.exclusionPolicy.markdownDescription%",
+            "enumDescriptions": [
+              "%c_cpp.configuration.exclusionPolicy.checkFolders.description%",
+              "%c_cpp.configuration.exclusionPolicy.checkFilesAndFolders.description%"
+            ],
+            "scope": "resource"
+          },
+          "C_Cpp.preferredPathSeparator": {
+            "type": "string",
+            "enum": [
+              "Forward Slash",
+              "Backslash"
+            ],
+            "default": "Forward Slash",
+            "markdownDescription": "%c_cpp.configuration.preferredPathSeparator.markdownDescription%",
+            "scope": "machine-overridable"
+          },
+          "C_Cpp.simplifyStructuredComments": {
```

Comment:
This and the next one are related to comments.  Maybe they should go with the doxygen settings and we rename that section to "Comments"?

Reply:
Yeah, that sounds good to me. Michelle had a comment on that too.